### DISCUSSION TITLE
Github Releases version is incorrect

### DISCUSSION THREAD

**Discussion by mkosir** [vercel/turborepo#1885] [Help] [answered]
Hey 👋 
not sure where exactly to post this, but noticed for quite some time now already, that Github release version (`v1.4.3`) is not showing the latest one  (`v1.4.6`) which makes it a bit confusing (also for tracking through changelog).

![image](https://user-images.githubusercontent.com/3073826/189111105-67c5fd08-2bd4-4b26-b4e5-1dd84800c079.png)

Keep up the awesome work!

**Comment by mehulkar** [comment, marked as answer]
This is not updated! https://github.com/vercel/turborepo/releases. We're still doing Github releases manually so sometimes it falls out of sync with npm registry releases. I think we're slowly working on making this better. Thanks for the ping!
